docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,824 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "739993c7",
+   "metadata": {},
+   "source": [
+    "# Protein Folding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d64f8162",
+   "metadata": {},
+   "source": [
+    "### Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bd209d1",
+   "metadata": {},
+   "source": [
+    "The structure and function of many natural and human-engineered\n",
+    "proteins is still only poorly understood. As a result, our understanding of\n",
+    "processes connected with protein folding, such as those encountered in\n",
+    "Alzheimer’s disease, vaccine development, and crop improvement\n",
+    "research, has remained limited.\n",
+    "\n",
+    "Unfolded polypeptides have a very large number of degrees of freedom\n",
+    "and thus an enormous number of potential conformations. For example, a\n",
+    "chain with 100 aminoacids has on the order of 10<sup>47</sup> conformations. In\n",
+    "reality, however, many proteins fold to their native structure within\n",
+    "seconds. This is known as Levinthal’s paradox [1].\n",
+    "\n",
+    "The exponential growth of potential conformations with chain length\n",
+    "makes the problem intractable for classical computers. In the quantum\n",
+    "framework, our resource-efficient algorithm scales linearly with\n",
+    "the number of amino acids N."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3110941f",
+   "metadata": {},
+   "source": [
+    "The goal of this work is to determine the minimum energy conformation of a protein. Starting from a random configuration, the protein's structure is optimized to lower the energy. This can be achieved by encoding the protein folding problem into a qubit operator and ensuring that all physical constraints are satisfied. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29deb98b",
+   "metadata": {},
+   "source": [
+    "For the problem encoding we use: \n",
+    "\n",
+    "- Configuration qubits: qubits that are used to describe the configurations and the relative position of the different beads\n",
+    "\n",
+    "- Interaction qubits: qubits that encode interactions between the different aminoacids\n",
+    "\n",
+    "For our case we use a tetrahydral lattice (diamond shape lattice) where we encode the movement through the configuration qubits (see image below). \n",
+    "\n",
+    "<img src=\"aux_files/lattice_protein.png\" width=\"300\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f571d4a8",
+   "metadata": {},
+   "source": [
+    "The Hamiltonian of the system for a set of qubits $\\mathbf{q}=\\{\\mathbf{q}_{cf}, \\mathbf{q}_{in}\\}$ is \n",
+    "\n",
+    "$$H(\\mathbf{q}) = H_{gc}(\\mathbf{q}_{cf}) + H_{ch}(\\mathbf{q}_{cf}) + H_{in}(\\mathbf{q}) $$\n",
```

Comment:
Does `H_{in}` depend on the entire set of qubits (as it is currently written) or only on `q_{in}`?

Reply:
Yes H_in is depended in the full qubit registed since it requires knowledge of the structure of the beads.

Reply:
Might I suggest to make this more explicit then? Something like this:
```suggestion
    "$$H(\\mathbf{q}) = H_{gc}(\\mathbf{q}_{cf}) + H_{ch}(\\mathbf{q}_{cf}) + H_{in}(\\mathbf{q}_{cf}, \\mathbf{q}_{in}) $$\n",
```

Reply:
Yes H_in is depended in the full qubit registed since it requires knowledge of the structure of the beads.